jrnl/override.py

```diff
@@ -0,0 +1,47 @@
+# import logging
+def apply_overrides(overrides: list, base_config: dict) -> dict:
+    """Unpack CLI provided overrides into the configuration tree.
+
+    :param overrides: List of configuration key-value pairs collected from the CLI
+    :type overrides: list
+    :param base_config: Configuration Loaded from the saved YAML
+    :type base_config: dict
+    :return: Configuration to be used during runtime with the overrides applied
+    :rtype: dict
+    """
+    config = base_config.copy()
+    for pairs in overrides:
+
+        key_as_dots, override_value = list(pairs.items())[0]
+        keys = key_as_dots.split(".")
```

Comment:
Can the dot notation resolve logic be a standalone function? This will make it easier to test.

Reply:
can do